src/material/menu/testing/menu-harness.ts

```diff
@@ -40,38 +40,57 @@ export class MatMenuHarness extends ComponentHarness {
     return coerceBooleanProperty(await disabled);
   }
 
-  /** Whether the menu is open. */
+  /** Gets a boolean promise indicating whether the menu is open. */
   async isOpen(): Promise<boolean> {
     return !!(await this._getMenuPanel());
   }
 
+  /** Gets a promise for the text of the menu's trigger element. */
   async getTriggerText(): Promise<string> {
     return (await this.host()).text();
   }
 
-  /** Focuses the menu and returns a void promise that indicates when the action is complete. */
+  /**
+   * Focuses the menu
+   * @return A promise that resolves when the action is complete.
```

Comment:
I wonder if we could just update our dgeni setup to add this return automatically for `async` functions when the `@return` is left blank.

cc @devversion

Reply:
Yes. This is definitely possible. Here is a prototype/example: https://github.com/angular/components/pull/17751